api/src/main/java/com/epam/pipeline/utils/GitUtils.java

```diff
@@ -23,6 +23,7 @@
 
     //regex for alphanumeric characters, underscore, dots and dash, allowing dash only in the middle
     private static final Pattern NAME_PATTERN = Pattern.compile("^[a-zA-Z0-9_.]+([-.][a-zA-Z0-9_]+)*$");
+    private static final String PROJECT_NAME_IN_URL_PATTERN = "(?<=/)[\\w]+(?=\\.git$)";
```

Comment:
since we use this pattern for replacing a name, we won't we be able to use `.` in name of repo. So I believe that we should also leave a comment here that we don't count on dots here because of our rule of naming from method `convertPipeNameToProject` or change a pattern to  `(?<=/)[\\w\\.]+(?=\\.git$)`